setup.cfg

```diff
@@ -18,10 +18,10 @@ package_dir =
     =src
 python_requires = >=3.6
 install_requires =
-    numpy==1.22.2
-    tensorboard==2.8.0
-    torch==1.12.1
-    protobuf==3.20.*
+    numpy>=1.22.2
+    protobuf<3.20,>=3.9.2
```

Comment:
```suggestion
```
See https://github.com/ami-iit/mann-pytorch/issues/2#issuecomment-1245041540 for a rationale.